terminologies/terminology/softwareFeatureCategory.jsonld

```diff
@@ -0,0 +1,8 @@
+{

```

Comment:
softwareFeatureCategory should be removed (for all: schemas, terminologies, etc) was depricated

Reply:
If it is depricated, why have the terms in the terminology-folder been updated by @jagru20 3 days ago? 

The schema for this has been deleted already and isn't used anywhere anymore. So, @lzehl is of course right. It is depricated. @jagru20, since you've done recent updates to this depricated terminology, do you need these terms to be moved to ("controlledTerms/terminologies/softwareFeature")? If you don't need them at all, can we delete the entire folder ("controlledTerms/terminologies/softwareFeatureCategory") with all its content?

Reply:
The only reason why I made changes (i.e., changing the case of name property to lower case) was that the folder was still existent in my fork of the project. It is in deed deprecated and can be deleted.

If someone reads this later and wonders; The decision not to use the softwareFeatureCategory was drawn in https://github.com/HumanBrainProject/openMINDS_core/issues/142

Reply:
Okay, then I'll delete the whole thing :)